cms/static/js/views/pages/container.js

```diff
@@ -195,8 +195,8 @@ define(['jquery', 'underscore', 'backbone', 'gettext', 'js/views/pages/base_page
                     (useNewVideoEditor === "True" && blockType === "video") ||
                     (useNewProblemEditor === "True" && blockType === "problem")
                 ) {
-                    var destinationUrl = this.$('.edit-button').attr("authoring_MFE_base_url") + '/' + blockType + '/' + this.$('.studio-xblock-wrapper').attr("data-locator");
-                    window.location.replace(destinationUrl);
+                    var destinationUrl = this.$('.edit-button').attr("authoring_MFE_base_url") + '/' + blockType + '/' + encodeURI(xblockElement.find('.xblock').attr("data-usage-id"));
+                    window.location.href = destinationUrl;
```

Comment:
Should this behavior be tested? Meaning, check the `data-usage-id` coming out of this function to confirm URI encoding? Or some more appropriate test?

Reply:
The change looks 👍🏼 otherwise.

Reply:
That's a great question. Do you mean that we want to confirm `data-usage-id` is encodable? the `encodeURI()` function encodes a URI by replacing each instance of certain characters by one, two, three, or four escape sequences representing the UTF-8 encoding of the character.  I'm not sure how it might not be encodable, as all attributes are strings, right?

Reply:
I meant just simply confirm that a non-uri-encoded `data-usage-id` ends up to be uri-encoded after this function executes. Maybe there's a better/more appropriate test? Just thinking of code coverage - I'll leave it up to you on whether you add a test here or not.

Reply:
And I *really* meant check `window.location.href` to see if it contains a uri-encoded link.